src/lib/ngsi10/NotifyContextRequest.cpp

```diff
@@ -61,6 +64,46 @@ std::string NotifyContextRequest::render(ConnectionInfo* ciP, RequestType reques
 
 /* ****************************************************************************
 *
+* NotifyContextRequest::toJson -
+*/
+std::string NotifyContextRequest::toJson(ConnectionInfo* ciP, NotificationFormat notifyFormat)
```

Comment:
...or maybe not in this case, as it seems that we have an old render() method that may need ciP (just after "First, v1 rendering of notification?").